### ISSUE TITLE
accept input in Apertium stream format

### ISSUE THREAD

**Issue by jonorthwash** [jonorthwash/ud-annotatrix#139]
Input pasted in from [Apertium stream format](http://wiki.apertium.org/wiki/Apertium_stream_format) should be accepted and converted to CoNLLU.

**Comment by ftyers** [comment]
Do we need this if we have CG input ? Also, it can't be converted (like CG) unless it's unambiguous.

**Comment by jonorthwash** [comment]
Yeah, let me put it in the long-term wishlist category.  It's by no means necessary, but it would still be nice :)

(Is opening Word format in libreoffice necessary if it can open rtf too?)
